python/mosaic/api/aggregators.py

```diff
@@ -8,7 +8,14 @@
 # Spatial aggregators #
 #######################
 
-__all__ = ["st_intersection_aggregate", "st_intersects_aggregate", "st_union_agg", "grid_cell_union_agg", "grid_cell_intersection_agg"]
+__all__ = [
+    "st_intersection_aggregate",
```

Comment:
Recommend we eventually standardize function naming to '_agg' vs some being '_aggregate'.

Reply:
I will add _agg aliases to avoid breaking changes

Reply:
aliases added.